### PR TITLE
Todo List: JS Best Practices

### PR DESCRIPTION
**Pull request by DammyShittu**
## In this pull request,

- Linters were properly set up
- Webpack was used
- Files were created as modules

##  Functionalities were added to;

- Add Todo Task
- Edit Todo Task
- Remove Todo Task
- Clear Completed Tasks
- Indexes were updated after deleting a task to represent the current list order.
- Preserve Data In Local Storage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  clearInput();
+  addToLocalStorage(todo);
+  showTasks();
+};
+
+const setIndex = (tasks) => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i + 1;
+  }
+};
```
Comment (ZeenatLawal) [inline src/index.js]: I think it'll be good to use `forEach` to update the index here, I believe this will increase your code readability.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
### Well Done @DammyShittu, Your repo looks professional👏🏼

### Change Required ♻️ 
- Please check the comment below for the suggested change

Happy Coding 😄 💻

File: src/index.js (hunk 1 above)
Comment (DammyShittu) [reply src/index.js]: Thank you for the review @ZeenatLawal 
I'll work on the requested changes now. 😃

**Review by ZeenatLawal — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @DammyShittu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

**Pull request merged**